nbpackage/src/main/resources/org/apache/netbeans/nbpackage/rpm/rpm.spec.template

```diff
@@ -0,0 +1,23 @@
+Name:           ${RPM_PACKAGE}
+Version:        ${RPM_VERSION}
+Release:        0
+Summary:        ${RPM_SUMMARY}
+License:        Apache License 2.0
+Group:          Development/Tools/IDE
+URL:            https://netbeans.apache.org/
```

Comment:
The default template needs to be software neutral.  Which fields are required?  If we need a license field, it needs to be an option, same with URL.  If group needed, can it be the same as the .desktop file group?

Reply:
I think that the required fields are the following ones:
- Name
- Version
- Release
- Summary
- License

The more information, the better :)

OK, license and URL as options.

The .desktop categories cannot be the same as the spec group; 
Categories=Development;Java;IDE; vs Group: Development/Tools/IDE

Reply:
> The more information, the better :)

In general, maybe.  However, we need to provide options for any additional fields - eg. `${RPM_LICENSE}` if needed.  Although that might be better as a global option as might be used elsewhere?

The DEB control file is just the minimum required to work well enough.  User can always override the whole template.  However, anything specific to NetBeans itself shouldn't be hardcoded.  NBPackage is not intended just for packaging NetBeans itself.

Reply:
I have added the following options: url, group, vendor and license.
License, url and group default to "Unknown", and vendor to maintainer.